README.md

```diff
@@ -9,4 +9,13 @@ https://devanswe.rs/create-application-specific-password-gmail/
 
 ```sh 
 docker compose up --build 
-```
\ No newline at end of file
+```
+
+# Possible Alerts: 
+
+- Slow Response Time: 95th percentile is higher than 10s 
+
+- Slow Database Query Time: 95th percentile is higher than 2,5s 
```

Comment:
You have 2 metrics that measure time of processing. `Slow Database Query` is somehow a subitem of `Slow Response Time`, because many requests require to make query to database. 

Why did you choose 4x difference between these values?

Reply:
Subjective assumption. Requests sometimes also include access to third-party APIs, which significantly increases the average response time